example/pom.xml

```diff
@@ -74,7 +74,17 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
       <resource>
         <directory>${project.basedir}/docker</directory>
         <filtering>true</filtering>
-        <targetPath>${project.build.directory}/hono</targetPath>
+        <targetPath>${project.build.directory}/hono/example/docker</targetPath>
```

Comment:
I was thing to leave it as it is, but this makes more sense to me, so I'm for it